ixofoundation/ixo-webclient
# Audricus 1 2 payment

## Scope
[WEB-1](https://ixo.youtrack.cloud/issue/WEB-1) Correct Display of Actions for Context and User Role
[WEB-2](https://ixo.youtrack.cloud/issue/WEB-2) No Live display of Claims Submitted on Explorer Cards

## Work Done
display actions by roles and remove hardcoded action and update actions icon as triangles
payment parts
project card in explore and change impact claims to headline metrics and added disputed status

## Steps to Test
payment: create template and copy template id and paste it in create contract and make payment
check dashboard actions is shown by user roles 

## Screenshots
